d2.config.js

```diff
@@ -2,7 +2,7 @@ const config = {
     type: 'app',
     title: 'Data Entry',
     pwa: {
-        enabled: process.env.REACT_APP_NODE_ENV !== 'test',
+        enabled: process.env.REACT_APP_NODE_ENV !== 'production',
```

Comment:
PWA caused some weird behavior from time to time during dev, we could revert this change before merging